chadwalt/andela-buddy-front
# 155932486: Enable users to sign-in with Andela Email

#### What does this PR do?
This PR enables logging into the application using an Andela email address and password.

#### Description of Task to be completed?
This PR enables users to login to the application using an Andela email address. It also adds sass and redux. It also allows automatic redirection to the login page if a user try's to access a protected `url link`.

#### How should this be manually tested?
Make sure you're not logged in the system and access http://localhost:3000/login

#### What are the relevant pivotal tracker stories?
[155932486](https://www.pivotaltracker.com/story/show/155932486)

#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/8415776/38241686-8b082770-373b-11e8-851c-713cbae568a4.png)
